### ISSUE TITLE
Multiple Card Hands for Different Decks

### ISSUE THREAD

**Issue by kristianserrano** [spacemandev-git/fvtt-card-support#33]
I have no idea if this is possible, but I was curious if a hand could be created for each deck in the system. I could imagine these hands being switched like the macro bars. Then if players have power cards, for example, they could flip through their hands to reference those cards.

**Comment by Norc** [comment]
This should be possible.

I know that Spaceman was intending that all cards be in one hand regardless of deck, but I think I like this idea better. And you are correct in that I could possibly implement a page system to make it work.

This would be a not-insignificat change, but it should be doable. Thoughts, Spaceman?

**Comment by spacemandev-git** [comment]
Multiple pages is better, players can use different pages to sort their hand as they see fit. If they want to do it by deck, they can do it by deck. If they want to do it some other way, they can do it some other way. We shouldn't mandate different hands = different decks

**Comment by kristianserrano** [comment]
Fair enough. Different hands could mean different characters for example (if the GM allows it).

**Comment by RhombusWeasel** [comment]
I can only speak for Deadlands Classic games but different hands for different decks would be ideal for that.  I have at least 3 decks to use for Fate Chips, Action Deck and Huckster Deck (and another if the Mad Scientist wants to do something!) and with no real way to determine which card came from which deck then player sorting would be less than ideal.  Maybe have it as a GM option?

**Comment by Norc** [comment]
Thanks for the feedback. Different size/visual appearance of cards can help in the meantime, but certainly having different hands would be an ideal solution.

**Comment by RhombusWeasel** [comment]
I've been using different deck graphics for the different decks to make it easier for people.  Is it worth considering the Roll20 approch? One hand but with separate divs for the different decks as you get dealt cards which should be easily sortable by deckId.
Also thanks for making this.  Without it I'd be unable to switch my Deadlands game over to Foundry, Spacemandev's youtube videos were invaluable when figuring out how to create my own module to extend the dice roller.  Thanks for the work you've put in, most people that use this won't be dev's so won't appreciate the effort that goes into making something that is as stable as this.  Great work.
